### DISCUSSION TITLE
Using different pins for DAC

### DISCUSSION THREAD

**Discussion by mrhempman69** [pschatzmann/ESP32-A2DP#481] [Q&A] [answered]
I am trying to use pins 34,35 for BCK and WS, but no audio comes out. I changed the pins in the config and flashed successfully.

When using the default pins, audio comes out but is noisy.

I am using the external DAC with the purple PCB and gold TRRS fingers from the wiki.

How might one succesfully use pins that are not associated with the internal DAC to drive the speakers?

I assumed it was possible by nature of the way the config has exposed these pins as modular options. So if this is currently possible for you and not for me, then it may be a hardware issue on my end.

**Comment by pschatzmann** [comment, marked as answer]
To my knowledge I2S can be used on all GPIO __output__ pins. 
https://randomnerdtutorials.com/esp32-pinout-reference-gpios/

So your input only pins will naturally not work...

**Comment by mrhempman69** [comment]
Thank you, that is true, it works on pins 32,35. Not sure how i got so unlucky picking input only pins
